hi-ml-cpath/testhisto/testhisto/utils/test_deepmil_utils.py

```diff
@@ -57,7 +56,6 @@ def test_load_ssl_checkpoint_from_url(tmp_path: Path) -> None:
     assert isinstance(encoder, SSLEncoder)
 
 
-@pytest.mark.skip(reason="This test is failing because of issue #655")
 def test_load_ssl_checkpoint_from_run_id(tmp_path: Path) -> None:
     encoder_params = EncoderParams(encoder_type=SSLEncoder.__name__, ssl_checkpoint=CheckpointParser(TEST_SSL_RUN_ID))
     assert encoder_params.ssl_checkpoint.is_aml_run_id
```

Comment:
do we still need to patch get_workspace line 62? I saw that you removed it in test_encoder? will it be inferred from env variables of the github agent?

Reply:
Very good point. Removed here and at several other places